sonar-ws/src/main/java/org/sonarqube/ws/client/BaseRequest.java

```diff
@@ -60,16 +68,79 @@ public SELF setMediaType(String s) {
     return (SELF) this;
   }
 
+  /**
+   * To set a multi value parameters, provide a Collection with the values
+   */
   public SELF setParam(String key, @Nullable Object value) {
```

Comment:
Why are we using Object ? Why not creating many methods with different classes ?